gapic/src/main/com/google/gapid/views/PipelineView.java

```diff
@@ -197,22 +200,36 @@ private void createPipelineTabs(TabFolder folder, List<API.Pipeline> pipelines)
         for (API.Stage stage : stages) {
           TabItem item = createStandardTabItem(folder, stage.getDebugName());
 
-          Group stageGroup = createGroup(folder, stage.getStageName());
+          Composite stageGroup = createComposite(folder, new GridLayout());
+
+          FillLayout nameLayout = new FillLayout(SWT.VERTICAL);
+          nameLayout.marginHeight = 5;
+          Composite nameComposite = withLayoutData( createComposite(stageGroup, nameLayout),
+              new GridData(SWT.BEGINNING, SWT.BEGINNING, false, false));
+
+          Label stageName = createBoldLabel(nameComposite, stage.getStageName() + " (" + stage.getDebugName() + ")");
```

Comment:
use `createLabel`, since `createBoldLabel` sets a font, which you then override again.